bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -330,7 +336,7 @@ spec:
           fi
 
           # start nbctl daemon for caching
-          export OVN_NB_DAEMON=$(ovn-nbctl --pidfile=/run/openvswitch/ovnk-nbctl.pid \
+          export OVN_NB_DAEMON=$(ovn-nbctl --pidfile=ovnk-nbctl.pid \
```

Comment:
the container's `preStop` hook still does `kill $(cat /run/openvswitch/ovnk-nbctl.pid)`

Reply:
fixed